### DISCUSSION TITLE
项目无法运行

### DISCUSSION THREAD

**Discussion by SuzukiAzusa** [nonebot/discussions#442] [问答]
小白一个，跟着教程走还是出错了sad
![image](https://github.com/nonebot/discussions/assets/160495378/3f1f5c95-6314-464c-8bfc-5a6a93a98dee)

**Comment by he0119** [comment]
这是 Console 适配器没有适配最新的 Pydantic V2 版本导致的。

你可以激活虚拟环境后，运行 `pip install --force-reinstall 'pydantic~=1.10'` 降级 Pydantic 版本。

**Reply by SuzukiAzusa** [reply]
![image](https://github.com/nonebot/discussions/assets/160495378/926f1c37-8ce8-47a8-84f2-032f0944359e)
已经降级了还是不行，和上面一模一样的错误QAQ

**Reply by StarHeartHunt** [reply]
降级的是虚拟幻境里的吗
